### PR TITLE
Print integration test log when case failed

### PR DESCRIPTION
**Pull request by Crazykev**
Signed-off-by: Crazykev <crazykev@zju.edu.cn>

### CODE REVIEW

**Comment by Crazykev** [comment]
@laijs PTAL
1. I've change print log logic to only print runv log when test case failed.
2. runv log always missing due to glog flush interval is 30s and can't change outside. Add a goroutine to flush glog in every 5s, and manually flush when reap pod and containerd.

File: integration-test/init_test.go [hunk 1]
```diff
@@ -23,6 +25,7 @@ const (
 	initrdName     = "hyper-initrd.img"
 	binaryName     = "runv"
 	rootfsName     = "rootfs"
+	logFileName    = "runv.INFO"
```
Comment (laijs) [inline integration-test/init_test.go]: the runv.ERROR is not printed in this patch.
if the test starts multiple runv daemon, it prints the last one only.

Comment (Crazykev) [inline integration-test/init_test.go]: INFO has already include WARNNING and ERROR, so I skipped them.
Updated to print all the INFO log in log_dir.

**Comment by laijs** [comment]
LGTM

**Pull request merged**